README.md

```diff
@@ -45,9 +44,8 @@ Jest will pick up tests in files with the `.itest.js` extension.
 1. `./scripts/make-shared-build-env.sh`: makes a shared base image containing all npm dependencies
 1. `./scripts/make-integration-test-image.sh` makes the integration tests image which contains all selenium test 
 dependencies
-1. `./scripts/run-build.sh`: creates and runs the main build env image which dockerises and pushes the app image 
-to docker hub
+1. `./scripts/build-image.sh`: builds and pushes the main app image to docker hub
 1. `./scripts/run-integration-tests.sh`: runs the app image created in the previous step along with all dependencies and 
 then runs the integration tests image created in step 2.
-1. `./dev/run-build-minimal.sh`: builds an image `montagu-reverse-proxy-minimal` that just provides login functionality.
+1. `./dev/build-minimal-image.sh`: builds an image `montagu-reverse-proxy-minimal` that just provides login functionality.
```

Comment:
Will it? `build-minimal-image.sh` references the `.` Dockerfile, which is provided in the `dev` folder along with the script, but if you're running that from the project root folder, won't it use the Dockerfile at the root? Might be better to give the dev (minimal) dockerfile a more explicit name like  `buildMinimal.dockerfile` like it used to have.